src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -158,4 +153,37 @@ export const TrackPanelList = () => {
   );
 };
 
+const FocusObject = (props: { focusObject: FocusObjectType | null }) => {
+  const { focusObject } = props;
+  let content: ReactNode;
+
+  if (focusObject?.type === 'gene') {
+    content = <FocusGene focusGene={focusObject} />;
+  } else if (focusObject?.type === 'variant') {
```

Comment:
Wonder if we should introduce `FocusObjectType` like `TrackType` and have these `gene` and `variant` as different focus object types.

Reply:
Enums are evil.

Typescript knows what types of the focus object are possible at this point in the code:

![image](https://user-images.githubusercontent.com/6834224/222088073-e5d4f77e-4939-48b3-9374-eca9ea94efd6.png)